### PR TITLE
Add sample config with cri job environment

### PR DESCRIPTION
**Pull request by koct9i**

### CODE REVIEW

File: config/samples/cluster_v1_cri.yaml [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+          cpu: 4
+          memory: 4Gi
+        limits:
+          cpu: 10
+          memory: 10Gi
+
+      jobEnvironment:
+        # Add CRI containerd sidecar
+        cri:
+          apiRetryTimeoutSeconds: 180
```
Comment (gritukan) [inline config/samples/cluster_v1_cri.yaml:282]: Do we need to customize it?

Comment (koct9i) [reply config/samples/cluster_v1_cri.yaml:282]: Right now - yes, some tuning is required.

**Comment by koct9i** [comment]
This config (with 1 replica of each component) depends on this change: https://github.com/ytsaurus/yt-k8s-operator/pull/185

**Review by gritukan — APPROVED** [review]

**Comment by gritukan** [comment]
Shipped both of them.

**Pull request merged**